cmd/nerdctl/image_convert.go

```diff
@@ -165,6 +180,41 @@ func imageConvertAction(cmd *cobra.Command, args []string) error {
 		}
 	}
 
+	if nydus {
+		if estargz {
+			return errors.New("option --nydus conflicts with --estargz")
+		}
+		if zstdchunked {
+			return errors.New("option --nydus conflicts with --zstdchunked")
+		}
+		if !oci {
+			logrus.Warnln("option --nydus should be used in conjunction with '--oci', forcibly enabling on oci mediatype for nydus conversion")
+		}
```

Comment:
Um... How does it forcibly enable oci option as the warning message says?

Reply:
The value of the `docker2oci` param in the `IndexConvertFuncWithHook` function will [be always set `true`](https://github.com/containerd/nerdctl/pull/1381/files#diff-54ec048a1d59c912fc54d9a2463beb5f84676ef7a2f93cebcbd9dced0cda9027R211).